cmd/rofl/deploy.go

```diff
@@ -413,6 +413,9 @@ func showProviderOffer(ctx context.Context, offer *roflmarket.Offer) {
 		offer.Resources.CPUCount,
 		float64(offer.Resources.Storage)/1024.,
 	)
+	if _, ok := offer.Metadata["note"]; ok {
```

Comment:
I would use namespaces for metadata, eg. `net.oasis.note` instead of a global "note".